cmake/load_package.cmake

```diff
@@ -21,7 +21,7 @@ macro(require_pkg_directory pkg_name pkg_user_name)
   endif()
 endmacro(require_pkg_directory)
 
-macro(find_package_local pkg_name pkg_directory pkg_other_name)
+macro(find_package_local pkg_name pkg_other_name)
```

Comment:
Do we really need the `pkg_other_name` parameter? I think we should be good with just searching for `Kokkos` and `KokkosKernels` is simply repeated anyways.

Reply:
For this to work we'd have to update all occurrences of `-Dkokkos_DIR` to `-DKokkos_DIR`. This would be breaking change, but I agree that it should be done.

I don't know the reason we're using lowercase version for Kokkos.

Reply:
We decided to use `-DKokkos_DIR`